js/main.js

```diff
@@ -1,10 +1,17 @@
+/* global data */
+/* exported data */
+
 var $form = document.querySelector('.form');
 var $resultsContainer = document.querySelector('.results-container');
 var $formContainer = document.querySelector('.form-container');
+var $listContainer = document.querySelector('.list-container');
 var $homeBtn = document.querySelector('.aw-logo');
 var $homeBtn2 = document.querySelector('.home');
 var $score = document.getElementById('score');
-
+var $listRow = document.querySelector('.list-row');
+var $watchListDesk = document.querySelector('.watch-list-desktop');
+var $watchListMobile = document.querySelector('.watch-list-mobile');
+var fullAnime = {};
```

Comment:
You should be creating this object in the function you use it in. If you need this data in another function, you likely should be making use of parameters and arguments.